src/legacy/status_im/multiaccounts/update/core.cljs

```diff
@@ -10,7 +10,7 @@
   [{:keys [db]}]
   (let [{:keys [name preferred-name display-name address]} (:profile/profile db)]
     {:json-rpc/call [{:method     "wakuext_sendContactUpdates"
-                      :params     [(or preferred-name display-name name) ""]
+                      :params     [(or preferred-name display-name name) "" ""]
```

Comment:
What's the meaning of passing this empty string?

Reply:
it just means do not update profile customization color
profile color is handled by `wakuext_setCustomizationColor`